pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.wdl

```diff
@@ -0,0 +1,113 @@
+version 1.0
+
+import "https://raw.githubusercontent.com/broadinstitute/warp/snSS2_first_wdls/pipelines/skylab/smartseq2_single_nucleus/SmartSeq2SingleNucleus.wdl" as single_nucleus_run
```

Comment:
relative imports here as well?

Reply:
Done.

Reply:
Did these changes maybe not get pushed up?

Reply:
I changed it back to run the workspace in terra. Now it is all relative imports.